ee/tabby-schema/src/schema/web_documents.rs

```diff
@@ -53,11 +54,11 @@ pub struct CreateCustomDocumentInput {
 #[derive(Validate, GraphQLInputObject)]
 pub struct SetPresetDocumentActiveInput {
     #[validate(regex(
-        code = "name",
+        code = "id",
         path = "*crate::schema::constants::WEB_DOCUMENT_NAME_REGEX",
-        message = "Invalid document name"
+        message = "Invalid document id"
     ))]
```

Comment:
regex validation is no longer necessary for `ID` type (it's opaque and shall only be meaningful to backend, no need to verify its content)